code/game/gamemodes/cult/cult_comms.dm

```diff
@@ -75,3 +78,116 @@
 	popup.set_content(text)
 	popup.open()
 	return 1
+
+/mob/living/proc/cult_master()
+	set category = "Cultist"
+	set name = "Assert Leadership"
+	pollCultists()
```

Comment:
I believe the poll itself yoinks the verb until ~60 seconds after the vote is over